gcloud/translate/client.py

```diff
@@ -0,0 +1,199 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Client for interacting with the Google Cloud Translate API."""
+
+
+import httplib2
+import six
+
+from gcloud._helpers import _to_bytes
+from gcloud.translate.connection import Connection
+
+
+ENGLISH_ISO_639 = 'en'
+"""ISO 639-1 language code for English."""
+
+
+def _zip_assert(values1, values2):
```

Comment:
Seems weird to pull this out and then only use it once. in the module... Can we just in-line this check?